src/sources/dataModel/taxonomic-service.json

```diff
@@ -125,26 +261,78 @@
                 "This service is rejected, because..."
             ]
         },
+        "additionalProperty": {
+            "type": "array",
+            "description": "Generic property for additional properties not covered by schema.org",
+            "items": {
+                "type": "object",
```

Comment:
Add the required and additionalProperties to this. Probably all fields are required

Reply:
Done